app/src/main/java/com/junkfood/seal/ui/theme/Theme.kt

```diff
@@ -60,6 +55,16 @@ fun SealTheme(
         )
         else this
     }
+    val view = LocalView.current
+    if(!view.isInEditMode){
+        val currentWindow = (view.context as? Activity)?.window
+        SideEffect {
+            (view.context as Activity).window.statusBarColor = android.graphics.Color.TRANSPARENT
+            (view.context as Activity).window.navigationBarColor = android.graphics.Color.TRANSPARENT
+            WindowCompat.getInsetsController(currentWindow!!, view).isAppearanceLightStatusBars =
```

Comment:
The fix that really works for the bug seems to be here

Reply:
Yes. By some reason, rememberSystemUiController.run was causing the UI not getting updated.